src/app/components/FatalErrorHandler/index.tsx

```diff
@@ -46,8 +46,8 @@ export function FatalErrorHandler(props: Props) {
             <Trans
               i18nKey="fatalError.instruction"
               t={t}
-              components={[<Anchor href="https://github.com/oasisprotocol/oasis-wallet-web/issues" />]}
-              defaults="A fatal error has occurred and Oasis Wallet must stop. Try refreshing the page and reopening your wallets to see if the issue persists. Please file an issue to our <0>github project</0> and attach the report below."
+              components={[<Anchor href="mailto:wallet@oasisprotocol.org" />]}
+              defaults="A fatal error has occurred and Oasis Wallet must stop. Try refreshing the page and reopening your wallets to see if the issue persists. Engage directly with our support team regarding issues via <0>wallet@oasisprotocol.org</0> email. Please attach the report below."
```

Comment:
A fatal error has occurred and Oasis Wallet must stop. Try refreshing the page and reopening your wallets to see if the issue persists. If the issue persists, please contact our support team via <0>wallet@oasisprotocol.org</0> email and attach the report below.